### ISSUE TITLE
Is there a way to provision exchanges, queues and bindings ?

### ISSUE THREAD

**Issue by gustavobgama** [saltstack-formulas/rabbitmq-formula#40]
Hi guys,

I am exploring the formula and I would know if is there a way to create **exchanges, queues and bindings**.

I tried to inform queue data at pillar to create a queue:

```
rabbitmq:
  (...)
  queues:
    queue_name:
      - vhost: '/'
      - durable: True
      - auto_delete: False
```

But, no success. Reading the formula code I found:

```
{% set module_list = salt['sys.list_modules']() %}
{% if 'rabbitmqadmin' in module_list %}
include:
  - .config_bindings
  - .config_queue
  - .config_exchange
{% endif %}
```

What is this **rabbitmqadmin** sys module ? How can I install it ? I tried to run the state **rabbit-management.sls** (it run successfully) but when I list sys modules (sudo salt-call sys.list_modules | grep rabbitmqadmin) it was not created as a sys module.

Any help will be much appreciated !

**Comment by aleksey-hariton-epam** [comment]
@gustavobgama did you found any solution?

**Comment by sylvainfaivre** [comment]
Has anyone succeeded to do this ? I think that these items require https://github.com/PKiman/saltstack_rabbitmq_extended but I can't get it to work.

**Comment by noelmcloughlin** [comment]
Included in #66
